libs/langchain-scripts/bin/build

```diff
@@ -601,7 +601,8 @@ const generateImportMap = ({
     "\n"
   )}\n${extraDeclarations.join("\n")}\n`;
 
-  if (!extraContent.trim()) {
+  extraContent.trim();
+  if (!/[a-zA-Z0-9]/.test(extraContent)) {
```

Comment:
the `!extraContent` check wasn't working as intended since the variable still had two newlines in it